libs/deal-ticket/src/components/trading-mode-tooltip/trading-mode-tooltip.tsx

```diff
@@ -0,0 +1,93 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { MarketTradingMode, AuctionTrigger } from '@vegaprotocol/types';
+import { ExternalLink } from '@vegaprotocol/ui-toolkit';
+import type { ReactNode } from 'react';
+import { MarketDataGrid } from './market-data-grid';
+
+type TradingModeTooltipProps = {
+  market: {
+    tradingMode: MarketTradingMode;
+    data: { trigger: AuctionTrigger | null } | null;
+  };
+  compiledGrid: { label: ReactNode; value?: ReactNode }[];
+};
+
+export const TradingModeTooltip = ({
+  market,
+  compiledGrid,
+}: TradingModeTooltipProps) => {
+  switch (market.tradingMode) {
+    case MarketTradingMode.TRADING_MODE_CONTINUOUS: {
+      return (
+        <>
+          {t(
+            'This is the standard trading mode where trades are executed whenever orders are received.'
+          )}
+        </>
+      );
+    }
+    case MarketTradingMode.TRADING_MODE_OPENING_AUCTION: {
+      return (
+        <>
+          <p className="mb-4">
+            <span>
+              {t(
+                'This new market is in an opening auction to determine a fair mid-price before starting continuous trading.'
+              )}
+            </span>{' '}
+            <ExternalLink href="https://docs.fairground.vega.xyz/docs/trading-questions/#auctions-what-happens-in-an-opening-auction">
```

Comment:
question (blocking?): shouldn't we eventually point to the `docs.vega.xyz`? Perhaps it'd be better if taken from env.

Reply:
I am not really sure. I did not update the links, just moved this whole file in another lib. @candida-d does it matter if we point to `docs.vega.xyz` or `docs.fairground.vega.xyz`? are they specific to an envirionment?

Reply:
@MadalinaRaicu docs.fairground is still relevant **but** you might as well change it to the new site now, as there's a new page covering auctions. https://docs.vega.xyz/docs/testnet/concepts/trading-on-vega/trading-modes#auction-type-opening

It is network-specific, the mainnet version will just be /mainnet/ instead of /testnet/

Good spot @asiaznik!